hub/core/chunk_engine/tests/common.py

```diff
@@ -119,9 +109,7 @@ def assert_chunk_sizes(
 def run_engine_test(
     arrays: List[np.ndarray], storage: StorageProvider, batched: bool, chunk_size: int
 ):
-    clear_if_memory_provider(storage)
-
-    key = random_key()
+    key = TENSOR_KEY
```

Comment:
In line 135:

Add #TODO for changing the code once meta load file code is merged.